actions/subscribers.go

```diff
@@ -369,17 +393,47 @@ func ImportSubscribers(c *gin.Context) {
 		})
 		return
 	}
+	res, err := client.GetObject(&s3.GetObjectInput{
+		Bucket: aws.String(os.Getenv("FILES_BUCKET")),
+		Key:    aws.String(fmt.Sprintf("subscribers/import/%d/%s", u.ID, body.Filename)),
+	})
+	if err != nil {
+		c.AbortWithStatusJSON(http.StatusInternalServerError, gin.H{
+			"message": "Unable to import subscribers. Please try again.",
+		})
+		return
+	}
+
+	csvCount, err := lineCounter(res.Body)
+	if err != nil {
+		c.AbortWithStatusJSON(http.StatusInternalServerError, gin.H{
+			"message": "Unable to import subscribers. Please try again.",
+		})
+	}
 
-	go func(ctx context.Context, client s3iface.S3API, filename string, userID int64, segs []entities.Segment) {
+	_, limitCount, err := boundariesSvc.SubscribersLimitExceeded(u)
```

Comment:
Fetch the limit count first and check for the exceeded flag, if it's true then there's no need to do the line counting.